NEWS.md

```diff
@@ -4,34 +4,42 @@ Julia v1.7 Release Notes
 New language features
 ---------------------
 
-* `(; a, b) = x` can now be used to destructure properties `a` and `b` of `x`. This syntax is equivalent to `a = getproperty(x, :a)`
-  and similarly for `b`. ([#39285])
-* Implicit multiplication by juxtaposition is now allowed for radical symbols (e.g., `x√y` and `x∛y`). ([#40173])
+* `(; a, b) = x` can now be used to destructure properties `a` and `b` of `x`.
+  This syntax is equivalent to `a = getproperty(x, :a); b = getproperty(x, :b)` ([#39285]).
+* Implicit multiplication by juxtaposition is now allowed for radical symbols (e.g. `x√y` and `x∛y`) ([#40173]).
 * The short-circuiting operators `&&` and `||` can now be dotted to participate in broadcast fusion
-  as `.&&` and `.||`. ([#39594])
+  as `.&&` and `.||` ([#39594]).
 * `⫪` (U+2AEA, `\Top`, `\downvDash`) and `⫫` (U+2AEB, `\Bot`, `\upvDash`, `\indep`)
-  may now be used as binary operators with comparison precedence. ([#39403])
-* Repeated semicolons may now be used inside array literals to separate dimensions of an array,
-  with the number of semicolons specifying the particular dimension. Just as the single semicolon
-  in `[A; B]` has always described concatenating along the first dimension (vertically), now two
+  may now be used as binary operators with comparison precedence ([#39403]).
+* Repeated semicolons can now be used inside array concatenation expressions to separate dimensions
+  of an array, with the number of semicolons specifying the dimension. Just as a single semicolon
+  in `[A; B]` has always described concatenating in the first dimension (vertically), now two
   semicolons `[A;; B]` do so in the second dimension (horizontally), three semicolons `;;;` in the
-  third, and so on. ([#33697])
+  third, and so on ([#33697]).
 * A backslash (`\`) before a newline inside a string literal now removes the newline while also
   respecting indentation. This can be used to split up long strings without newlines into multiple
-  lines of code. ([#40753])
+  lines of code ([#40753]).
+* A backslash before a newline in command literals now always removes the newline, similar to standard string
+  literals, whereas the result was not well-defined before ([#40753]).
```

Comment:
Is this really a language feature?

Reply:
I would say so since it is all the way into the parser.

Reply:
The cmd literal part actually wasn't a change in Julia's parser, but only in `shell_parse`. Julia's parser only strips backslash+newline for non-custom string literals.

Reply:
Oh, I think I misread this, sorry.

Reply:
I think it will strike people as a syntax change.

Reply:
Also wanted to put it next to the backslashes-in-strings change, since they are similar.

Reply:
It might be a bit of a gotcha within macros, but that's probably not a big issue. Seems fine like this to me then.